From 3d66cbd755b0ad77f35f2bc45f236b25f7854f25 Mon Sep 17 00:00:00 2001
From: Harsh J <harsh@cloudera.com>
Date: Tue, 26 Mar 2013 07:52:08 +0530
Subject: [PATCH 1303/1357] MR1: CLOUDERA_BUILD. Fix regression introduced by b5eaf8099 that makes a certain JobTracker limit configurable by clients if un-finalized.

Description: The JT's split meta info limits were being taken from client configs due to an earlier regression change. This fixes the bad behavior.
Reason: Customer Request
Author: Harsh J
Ref: CDH-10222
---
 .../org/apache/hadoop/mapred/JobInProgress.java    |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/hadoop-mapreduce1-project/src/mapred/org/apache/hadoop/mapred/JobInProgress.java b/hadoop-mapreduce1-project/src/mapred/org/apache/hadoop/mapred/JobInProgress.java
index ff39f57..202ac31 100644
--- a/hadoop-mapreduce1-project/src/mapred/org/apache/hadoop/mapred/JobInProgress.java
+++ b/hadoop-mapreduce1-project/src/mapred/org/apache/hadoop/mapred/JobInProgress.java
@@ -826,7 +826,7 @@ public class JobInProgress {
   TaskSplitMetaInfo[] createSplits(org.apache.hadoop.mapreduce.JobID jobId)
   throws IOException {
     TaskSplitMetaInfo[] allTaskSplitMetaInfo =
-      SplitMetaInfoReader.readSplitMetaInfo(jobId, fs, conf, jobSubmitDir);
+      SplitMetaInfoReader.readSplitMetaInfo(jobId, fs, jobtracker.getConf(), jobSubmitDir);
     return allTaskSplitMetaInfo;
   }
 
-- 
1.7.0.4

